modules/gridmap/editor/grid_map_editor_plugin.cpp

```diff
@@ -907,10 +907,12 @@ void GridMapEditor::update_palette() {
 			continue;
 		}
 
+		String tooltip_text = name + " (Item ID: " + itos(id) + ")";
```

Comment:
```suggestion
		String tooltip_text = vformat(TTR("%s (Item ID: %d)"), name, id);
```
Or similar, I'd say this should be translated